numpy/linalg/tests/test_regression.py

```diff
@@ -56,11 +56,6 @@ def test_svd_build(self):
 
         assert_array_almost_equal(b, np.zeros((2, 2)))
 
-    def test_norm_vector_badarg(self):
-        # Regression for #786: Froebenius norm for vectors raises
```

Comment:
This is strange: #786 doesn't seem to be a meaningful referral. I checked back in the PRs/Issues on github, and issue #786 has nothing to do with norms. It seems a lot of issues/PRs from this era were created with a different system, so this does not seem to be particularly helpful in resolving my above comment.

Either way, I agree this test should be removed if we go forward with the current approach

Reply:
This is why I've been leaning towards `gh-NNN` style referencing, so that if this happens again we're less confused.

Reply:
You're looking for https://github.com/numpy/numpy/issues/1384

Reply:
Thanks @eric-wieser . The conversation there doesn't appear to have any definitive conclusions, just noting that this issue was discovered way back in 2008